discovery.md

```diff
@@ -101,18 +101,295 @@ be considered an event subscriber and a target user of this API.
 
 ## API Specification
 
-This document structure focuses on discovery where each document has a two part
-key of `provider` and `type`. This takes a denormalized view where the composite
-key of documents is the `provider` and `type`. Provider is defined as an
-arbitrary human readable string and `type` is the CloudEvent `type` attribute.
+This API is specified as a REST API with well defined entities and relationships
+between those entities.
 
-### Provider entity Attributes
+All responses will have a content-type of `application/json` and are either
+a single JSON encoded object or an array of objects.
```

Comment:
For interop I agree with the idea of requiring JSON but I wonder if we should say that other formats are allowed via the normal HTTP `accept` negotiation stuff.  And absence of an `accept` defaults to JSON

Reply:
+1